tests/function-controller/testsuite/gitops/gitrepository.go

```diff
@@ -7,3 +7,10 @@ func NoAuthRepositorySpec(url string) serverlessv1alpha1.GitRepositorySpec {
 		URL: url,
 	}
 }
+
+func AuthRepositorySpecWithURL(url string, repoAuth *serverlessv1alpha1.RepositoryAuth) serverlessv1alpha1.GitRepositorySpec {
```

Comment:
`RepositorySpec`? I think it's enough.

Reply:
I am not sure I understand? This is an external type that I am using, I am not defining it.

Reply:
I mean, rename it to something like: `CreateGitRepoSpec`

Reply:
it's consistent with the existing function  https://github.com/kyma-project/kyma/pull/12824/files#diff-5576881242e9ac4029b9741f8459b61cf50c8967d92940ef267b33b7e1f8fe9cR5